testing/testing.test.js

```diff
@@ -1,25 +1,50 @@
-const { fail } = require('assert');
+const supertest = require('supertest');
+const mongoose = require('mongoose');
+
 const seedEntry = require('../database/seedCreation.js');
 const db = require('../database/index.js');
+const app = require('../server/index.js');
+
+const database = 'fecReviews';
+const request = supertest(app);
+
+beforeAll(async () => {
```

Comment:
I like this afterAll/beforeAll stuff, setting up/cleaning the DB on my todo list for my tests but I wasn't sure how to do it. Good idea